pkg/gadgets/trace/network/tracer/tracer.go

```diff
@@ -113,7 +119,7 @@ func (t *Tracer) Attach(key string, pid uint32) (err error) {
 	}
 
 	consts := map[string]interface{}{
-		"container_quark": t.nextContainerQuark,
+		"container_quark": netns,
```

Comment:
```suggestion
		"current_netns": netns,
```
Would that be a better name for the variable in the ebpf program?